pkg/component/etcd/statefulset/statefulset_test.go

```diff
@@ -422,12 +422,12 @@ func checkStatefulset(sts *appsv1.StatefulSet, values Values) {
 			"Name":      Equal(values.Name),
 			"Namespace": Equal(values.Namespace),
 			"Annotations": MatchAllKeys(Keys{
-				"checksum/etcd-configmap":   Equal("abc123"),
-				"gardener.cloud/owned-by":   Equal(fmt.Sprintf("%s/%s", values.Namespace, values.Name)),
-				"gardener.cloud/owner-type": Equal("etcd"),
-				"app":                       Equal("etcd-statefulset"),
-				"role":                      Equal("test"),
-				"instance":                  Equal(values.Name),
+				"checksum/etcd-configmap": Equal("abc123"),
```

Comment:
These changes are completely unrelated to etcdcopybackuptask and should not be clubbed with this PR. Please remove this and file a separate PR for this.